code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -254,18 +254,24 @@ Doesn't work on other aliens/AI.*/
 /obj/effect/proc_holder/alien/resin/fire(mob/living/carbon/user)
 	if(locate(/obj/structure/alien/resin) in user.loc)
 		to_chat(user, "<span class='danger'>There is already a resin structure there.</span>")
-		return 0
+		return FALSE
+
+	var/atom/movable/atmos_thing = locate(/obj/machinery/atmospherics/components/unary) in user.loc
```

Comment:
could be simplified to `var/obj/machinery/atmospherics/components/unary/atmos_thing = locate() in user.loc`

Reply:
that seems to be saying the same thing, just differently.

Reply:
Just a touch cleaner imo